o/opentelemetry-js/build_info.json

```diff
@@ -2,14 +2,14 @@
     "package_name" : "opentelemetry-js",
     "github_url":"https://github.com/open-telemetry/opentelemetry-js",
     "version": "v1.8.0",
-    "package_dir": "o/opentelemetry-js/latest",
+    "package_dir": "o/opentelemetry-js",
     "docker_build": false,
     "build_script" : "opentelemetry-js_ubi_8.5.sh",
-    
+
```

Comment:
@Pratikt2312 Same comment as I mentioned for opentelemtry-go.
Here you can use different node version

Reply:
Here we use the same node version for latest version v1.8.0 and its working fine, so change is not required.